network/discovery.go

```diff
@@ -246,22 +264,103 @@ func (d *discovery) findPeersCall(peerID peer.ID) ([]*peer.AddrInfo, error) {
 }
 
 func (d *discovery) run() {
+	ticker1 := time.NewTicker(peerDiscoveryInterval)
+	ticker2 := time.NewTicker(bootnodeDiscoveryInterval)
```

Comment:
These tickers should have more descriptive names, like `peerDiscoveryTicker` and `bootnodeDiscoveryTicker`

Reply:
Commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/aa333f9fcf309b004f3c1fbad82398cc7118a3b7)